### PR TITLE
Ucr geom, sdg layer options, dlg plot, ucr geom list with aes : partially mandatory x and y (default quotes assignment) ++ dlgScatterPlot and dlgBarChart quotes method bugfix

### PR DESCRIPTION
**Pull request by FrancoisJRenaud**
In progress

### CODE REVIEW

**Comment by dannyparsons** [comment]
How is this looking now?

**Comment by FrancoisJRenaud** [comment]
Yup, about to sync the last commit. VB crashed again ... However found quite a few bugs while testing. Will start a new issue/pull request about these...

**Comment by FrancoisJRenaud** [comment]
I think I m happy with this now. If you just want to have a quick look at comment in ucrGeomListWithAes, will write an issue about that bug and another one.

File: instat/UcrGeomListWithAes.vb [hunk 1]
```diff
@@ -115,18 +115,18 @@ Public Class UcrGeomListWithParameters
...
-                        If Not (clsParam.strArgumentName = "x" AndAlso clsParam.strArgumentValue = Chr(34) & Chr(34)) Then
+                        'For some geoms like LinePlot, when the x or y aes is not filled, ggplot R syntax requires to set x="". This x="" might be copied into the global aes if the ApplyOnAllLayers is set to true for a BoxPlot Layer. This might be copied from the GgplotAesFunction parameters into the aes receivers by error in subsequent layers.
+                        If Not ((clsParam.strArgumentName = "x" OrElse clsParam.strArgumentName = "y") AndAlso clsParam.strArgumentValue = Chr(34) & Chr(34)) Then
                             lstAesParameterUcr(i).Add(clsParam.strArgumentValue)
-                            lstAesParameterUcr(i).Enabled = Not bFixAes
+                            lstAesParameterUcr(i).Enabled = Not bFixAes 'Warning/Question/Task: this is not flexible enough. Some of the aesthetics are set in the options. They cannot be editted on the main, however when coming back to options these are fixed and so cannot be editted anywhere anymore. Would need to be able to choose which aesthetics among a Layer should be fixed maybe.
```
Comment (FrancoisJRenaud) [inline instat/UcrGeomListWithAes.vb]: Am creating an issue about this

**Pull request merged**